### PR TITLE
Cache - Try to use single ClusterRole binding for the deployer

### PR DESCRIPTION
**Pull request by Ark-kun**

### CODE REVIEW

File: manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/cache.yaml [hunk 1]
```diff
@@ -37,8 +37,17 @@ rules:
...
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - get
+  - patch
 ---
-apiVersion: rbac.authorization.k8s.io/v1
+{{- /*
```
Comment (rmgogogo) [inline manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/cache.yaml:49]: how about simply delete these two?

**Comment by rmgogogo** [comment]
can this PR work? Or just try?

If just try, Kustomize path may be cheaper to take a try.
I tested Kustomize path and it can't work. Better we fix that first which is much easier to solve than via MKP path. Testing a MKP path installation is painful and time consuming.

**Comment by Ark-kun** [comment]
Closing this experimental PR since the cache deploer seems to work fine as it is.

**Pull request closed without merging**